app/controllers/arrangement/WhatIsTheExpectedValueOfThisArrangementController.scala

```diff
@@ -71,10 +71,16 @@ class WhatIsTheExpectedValueOfThisArrangementController @Inject()(
       renderer.render("arrangement/whatIsTheExpectedValueOfThisArrangement.njk", json).map(Ok(_))
   }
 
+  private[arrangement] def stripWhiteSpace(body: AnyContent): Map[String, Seq[String]] =
```

Comment:
@iservport The trim should not happen here, it should be a form provider/ stringFormatter